app/views/instructor/courses/show.html.erb

```diff
@@ -40,7 +40,7 @@
     <%= link_to 'Add a new section...', new_instructor_course_section_path(current_course), class: 'btn btn-primary btn-lg' %>
   </div>
 
-  <ul>
+  <ul class="sections">
```

Comment:
You forgot to add the CSS class of `sections`.  This gets hooked up in the JavaScript higher in the file.